addons/web/static/src/views/pivot/pivot_arch_parser.js

```diff
@@ -37,23 +38,30 @@ export class PivotArchParser {
                     let fieldName = node.getAttribute("name"); // exists (rng validation)
 
                     archInfo.fieldAttrs[fieldName] = {};
-                    if (node.hasAttribute("string")) {
-                        archInfo.fieldAttrs[fieldName].string = node.getAttribute("string");
-                    }
                     if (
                         node.getAttribute("invisible") === "True" ||
                         node.getAttribute("invisible") === "1"
                     ) {
                         archInfo.fieldAttrs[fieldName].isInvisible = true;
                         break;
                     }
+                    for (const { name, value } of node.attributes) {
+                        if (["name", "type", "operator", "interval"].includes(name)) {
+                            continue;
+                        }
+                        if (name === "options") {
+                            archInfo.fieldAttrs[fieldName].options = evaluateExpr(value);
+                        } else if (name === "widget") {
+                            archInfo.widgets[fieldName] = value;
+                        }
+                        else if (!name.startsWith("t-att") && !name.startsWith("decoration-")) {
```

Comment:
I don't think we should consider those attributes. The pivot view does not accept owl directives or decorations.